extensions/wikia/Search/classes/Services/EpisodeEntitySearchService.php

```diff
@@ -45,23 +46,35 @@ protected function prepareQuery( string $query ) {
 
 		$select->createFilterQuery( 'ns' )->setQuery( '+(ns:(' . implode( ' ', $namespaces ) . '))' );
 		if ( in_array( strtolower( $slang ), static::$ARTICLE_TYPES_SUPPORTED_LANGS ) ) {
-			$select->createFilterQuery( 'type' )->setQuery( '+(article_type_s:' . static::EPISODE_TYPE . ' OR ' . static::EXACT_MATCH_FIELD . ':*)' );
+			$select->createFilterQuery( 'type' )->setQuery(
+				'+(article_type_s:' . static::EPISODE_TYPE . ' OR ' . static::EXACT_MATCH_FIELD . ':*)'
+			);
 		}
 
-		$dismax->setQueryFields( implode( ' ', [
-			'title_em^8',
-			'titleStrict',
-			$this->withLang( 'title', $slang ),
-			$this->withLang( 'redirect_titles_mv', $slang ),
-			static::EXACT_MATCH_FIELD . "^10",
-		] ) );
-		$dismax->setPhraseFields( implode( ' ', [
-			'title_em^8',
-			'titleStrict^8',
-			$this->withLang( 'title', $slang ) . '^2',
-			$this->withLang( 'redirect_titles_mv', $slang ) . '^2',
-			static::EXACT_MATCH_FIELD . "^10",
-		] ) );
+		$dismax->setQueryFields(
+			implode(
+				' ',
+				[
+					'title_em^8',
+					'titleStrict',
+					$this->withLang( 'title', $slang ),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "title" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)